app/src/main/java/com/solux/flory/presentation/home/HomeFragment.kt

```diff
@@ -4,20 +4,67 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
 import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
 import com.solux.flory.R
+import com.solux.flory.databinding.FragmentHomeBinding
+import com.solux.flory.presentation.home.HomeViewModel
 
 class HomeFragment : Fragment() {
-
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-    }
+    private lateinit var viewModel: HomeViewModel
```

Comment:
```suggestion
    private val viewModel by viewModels<HomeViewModel>()
```
뷰모델은 이런식으로 선언해주기!! 이렇게 선언하면 뷰모델 프로바이더 쓸 필요가 없어욤